geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/protobufs/TupleProtoBuf.scala

```diff
@@ -61,4 +96,15 @@ trait TupleProtoBuf {
       (spaceTimeKeyProtoBufCodec.decode(message.spaceTimeKey.get),
         multibandTileProtoBufCodec.decode(message.tiles.get))
   }
+
+  implicit def tupleSpaceTimeKeyImageProtoBufCodec =
+    new ProtoBufCodec[(SpaceTimeKey, Array[Byte]), ProtoTuple] {
+    def encode(tuple: (SpaceTimeKey, Array[Byte])): ProtoTuple =
+      ProtoTuple(
+        spaceTimeKey = Some(spaceTimeKeyProtoBufCodec.encode(tuple._1)),
+        imageBytes = ByteString.copyFrom(tuple._2))
+
+    def decode(message: ProtoTuple): (SpaceTimeKey, Array[Byte]) =
+      throw new Exception("This ProtoBuf message cannot be decoded")
```

Comment:
This could be decoded easily actually. Somebody will hit this, best to say its not implemented rather that it isn't done.